llvm/lib/Transforms/Instrumentation/SanitizerCoverage.cpp

```diff
@@ -724,7 +731,10 @@ GlobalVariable *ModuleSanitizerCoverage::CreateFunctionLocalArrayInSection(
   ArrayType *ArrayTy = ArrayType::get(Ty, NumElements);
   auto Array = new GlobalVariable(
       *CurModule, ArrayTy, false, GlobalVariable::PrivateLinkage,
-      Constant::getNullValue(ArrayTy), "__sancov_gen_");
+      Constant::getNullValue(ArrayTy),
```

Comment:
I don't understand why this change is needed? Also please avoid hardcoded AS200 checks and use DL.isFatPointer instead. But I am not convinced this change is actually required.